app/controllers/application_controller.rb

```diff
@@ -1,4 +1,17 @@
 # frozen_string_literal: true
+# Base controller class
```

Comment:
As in, we've made changes to the same file in both branches, even if some of those changes are identical.